Menu/GameShell/10_Settings/Storage/__init__.py

```diff
@@ -27,7 +28,7 @@ class StoragePage(Page):
     _BGmsg    = "%.1fGB of %.1fGB Used"
     _DskUsg   = None
 
-    _HighColor = pygame.Color(51,166,255)
+    _HighColor = pygame.Color(*SkinManager().GiveColor('High'))
```

Comment:
because the colors are shared against all skin managers we can shortcut the usage here

Reply:
I got you idea
you just want to move SkinManager out of every Page class